_maps/RandomRuins/LavaRuins/skyrat/lavaland_surface_arena.dmm

```diff
@@ -593,13 +581,11 @@
 /area/lavaland/surface/outdoors)
 "Dc" = (
 /obj/structure/stone_tile/slab,
-/obj/structure/necropolis_arch,
 /obj/structure/necropolis_gate{
 	desc = "A massive stone gateway into a gladitorial arena. You feel like entering this unprepared would be your doom."
 	},
-/turf/open/lava/smooth/lava_land_surface{
-	slowdown = 0
-	},
+/obj/structure/necropolis_arch,
+/turf/open/indestructible/boss,
```

Comment:
fun fact, the arch was ALSO runtiming because it was spawning on lava. gotta love it